lib/recording/RestoreRecordingFromBackup.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Created by lilach.maliniak on 05/11/2017.
+ */
+const Q = require('q');
+const fs = require('fs');
+const util = require('util');
+const events = require("events");
+const qio = require('q-io/fs');
+const path = require('path');
+const _ = require('underscore');
+const glob = require('glob');
+const config = require('../../common/Configuration')
+//config.set('logToConsole', true);
+const ErrorUtils = require('../utils/error-utils');
+const PersistenceFormat = require('../../common/PersistenceFormat.js');
+const logger = require('../../common/logger').getLogger("RestoreRecordingFromBackup");
```

Comment:
the usage of the logger is not good here, since we wnt to display to console + we don't want to write into the logs of the actual live controller